plugins/modules/cloudwatchlogs_log_group_metric_filter.py

```diff
@@ -56,7 +56,20 @@
         default_value:
           description:
             - The value to emit when a filter pattern does not match a log event.
+            - The I(default_value) and I(dimensions) options are mutually exclusive.
           type: float
+        unit:
+          description:
```

Comment:
Can you please add `version_added: 8.3.0` for each of these two new options?

Reply:
Done